src/main/java/net/glowstone/entity/GlowTntPrimed.java

```diff
@@ -24,6 +24,14 @@
     private int fuseTicks;
     private Entity source;
 
+    /**
+     * The player who ignited the TNT.
+     * null if not ignited by player.
+     */
+    @Getter
+    @Setter
+    private GlowPlayer player;
```

Comment:
Why are this variable and getter/setter necessary? The `getSource()` method should return the player who ignited the TNT (if any), according to docs.

Reply:
IOW:
```
public GlowPlayer getPlayer() {
    Entity source = getSource();
    return (source instanceof GlowPlayer) ? (GlowPlayer) source : null;
}
```